scripts/plant_growth_form.py

```diff
@@ -0,0 +1,105 @@
+#retriever
+from pkg_resources import parse_version
+
+from retriever.lib.models import Table
+from retriever.lib.templates import Script
+
+try:
+    from retriever.lib.defaults import VERSION
+
+    try:
+        from retriever.lib.tools import open_fr, open_fw, to_str
+    except ImportError:
+        from retriever.lib.scripts import open_fr, open_fw
+except ImportError:
+    from retriever import open_fr, open_fw, VERSION
+
+
+class main(Script):
+    def __init__(self, **kwargs):
+        Script.__init__(self, **kwargs)
+        self.name = "plant-growth-form"
+        self.title = "Plant Growth Form"
+        self.ref = "http://onlinelibrary.wiley.com/doi/10.1002/ecy.1569/full"
+        self.urls = {
+            "plantgrowth": "https://esajournals.onlinelibrary.wiley.com"
+                           "/action/downloadSupplement?doi=10.1002%2Fecy."
+                           "1569&attachmentId=184128600"
+        }
+        self.citation = "Engemann, K., Sandel, B., Boyle, B. L., Enquist, " \
+                        "B. J., Jorgensen, P. M., Kattge, J., McGill, " \
+                        "B. J., Morueta-Holme, N., Peet, R. K., Spencer, " \
+                        "N. J., Violle, C., Wiser, S. K. and Svenning, " \
+                        "J. .-C. (2016), " \
+                        "A plant growth form dataset for the New World. " \
+                        "Ecology. Accepted Author Manuscript. " \
+                        "doi:10.1002/ecy.1569"
+        self.tags = ['Taxon > Plants', 'Data Type > Observational']
+        self.retriever_minimum_version = "2.0.dev"
+        self.version = '1.0.0'
+        self.description = "Growth form classifications for 67,413 vascular " \
+                           "plant species from North, Central, and " \
+                           "South America."
+        current_version = parse_version(VERSION).__str__()
+        if current_version <= parse_version("2.0.dev"):
```

Comment:
changing this to
```
if current_version <= parse_version("2.0.dev").__str__():
```
made the script run for me, otherwise though, I think this script is good to go in.